# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/gcc4x.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

An explicit limits.h include is required for some new combination
of gcc/glibc.

*** linux-2.6.21/scripts/mod/sumversion.c.orig	2009-06-05 10:39:55.000000000 -0600
--- linux-2.6.21/scripts/mod/sumversion.c	2009-06-05 10:42:41.000000000 -0600
***************
*** 7,12 ****
--- 7,15 ----
  #include <ctype.h>
  #include <errno.h>
  #include <string.h>
+ #ifndef _LINUX_LIMITS_H
+ #include <limits.h>
+ #endif
  #include "modpost.h"
  
  /*
